AISKU/src/AISku.ts

```diff
@@ -152,6 +159,9 @@ export class AppInsightsSku implements IApplicationInsights {
 
             // Will get recalled if any referenced values are changed
             _addUnloadHook(onConfigChange(cfgHandler, () => {
+                if (!_cfgSyncPlugin){
```

Comment:
nit: As we are basically "setting" (creating a new one every 1st instance) then we can probably just move this into the `_initDefaults()` (no if required) just always create a new instance `_cfgSyncPlugin = new CfgSyncPlugin()` in there.